api/src/functionalTest/java/uk/gov/hmcts/payment/functional/tokens/ServiceTokenFactory.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.payment.functional.tokens;
+
+
+import io.restassured.parsing.Parser;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.TestComponent;
+import uk.gov.hmcts.payment.functional.IntegrationTestBase;
+
+import static io.restassured.RestAssured.*;
+
+@TestComponent
+public class ServiceTokenFactory extends  IntegrationTestBase {
```

Comment:
please don't use the base classes for utility concerns. Can the integrationTestBase be a utility class